net-analyzer/nrpe/nrpe-4.0.2-r1.ebuild

```diff
@@ -0,0 +1,105 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit autotools systemd
```

Comment:
We like to write one empty line below EAPI but it's not as important. Just do it if you edit this ebuild due to selinux comment.